Internship2022TeamPatri/Internship2022TeamPatri.xcodeproj/project.pbxproj

```diff
@@ -533,9 +549,10 @@
 				INFOPLIST_FILE = Internship2022TeamPatri/Info.plist;
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchStoryboardName = LaunchScreen;
-				INFOPLIST_KEY_UIMainStoryboardFile = Main;
+				INFOPLIST_KEY_UIMainStoryboardFile = Home;
```

Comment:
This should not be changed. The main entry point should still be Main storyboard, and on that one you need to implement a link to your Home storyboard, because if the user is not logged in, we will show the login screen not Home.